src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml

```diff
@@ -12,8 +12,6 @@
             <RowDefinition Height="Auto" />
             <RowDefinition Height="Auto" />
             <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
```

Comment:
Removed the two rows in the options grid corresponding to the checkbox and text block.